src/components/taskDetails/index.tsx

```diff
@@ -352,6 +352,12 @@ const TaskDetails: FC<Props> = ({ taskID }) => {
                                             data-testid="request-task-button"
                                             className={classNames.button}
                                             onClick={taskRequestHandle}
+                                            disabled={
+                                                !Object.is(
```

Comment:
This equality check will only check for the object to be null or not, in case it is undefined which we might get in the case when assignee is not present, or any other falsy value this equality check will return `true`